code/src/Styles/Button.css

```diff
@@ -0,0 +1,33 @@
+button {
+	width: 45%;
```

Comment:
In general your css files are very readable thanks to the fact that you have divided them into different files.